src/universal/components/TaskList/TaskList.js

```diff
@@ -31,6 +31,8 @@ const exampleItems = [
 ];
 
 @cssModules(styles)
+// for the decorators
+// eslint-disable-next-line react/prefer-stateless-function
 export default class TaskList extends Component {
   render() {
     const handleCheckboxChanged = (id) => {
```

Comment:
Nested functions! Extremely strange that this anti-pattern was used. 😕 

I just blindly updated the syntax.

Per my discussion with @jordanh, we're gonna push fixing this to a new PR.